.github/workflows/tests.yml

```diff
@@ -0,0 +1,74 @@
+name: Check stubs
+
+on:
+  push:
+  pull_request:
+
+jobs:
+  file-consistency:
+    name: Check file consistency
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        with:
+          python-version: 3.8
+      - run: ./tests/check_consistent.py
+
+  flake8:
+    name: Lint with flake8
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        with:
+          python-version: 3.8
+      - run: |
+          pip install $(grep flake8 requirements-tests-py3.txt)
+          flake8
+
+  black:
+    name: Check formatting with black
```

Comment:
It would be nice if we could also use actions to automatically fix isort and black formatting issues. From https://github.com/cclauss/autoblack it sounds like that may not be possible though.

Reply:
Definitely. There was some discussion in #3058.